lib/rijksbot.rb

```diff
@@ -9,35 +9,35 @@ def initialize(twitter_api)
   end
 
   def already_replied?(tweet)
-    already_replied_to =  @twitter_api.user_timeline(count: 800).map{|t| t.in_reply_to_status_id}
+    already_replied_to = @twitter_api.user_timeline(count: 800).map(&:in_reply_to_status_id)
     already_replied_to.include?(tweet.id)
   end
 
   # create def count_twitter_max = 800
   # create def one_day_in_seconds = 86400
 
   def extract_search(tweet)
-    tweet.text.downcase.sub("@rijks_bot", "").strip
+    tweet.text.downcase.sub('@rijks_bot', '').strip
   end
 
   def random_image
-    random_words = %w(lemon pineapple banana cheese orange butterfly stilleven vanitas painting ballet unicorn heart naakt dieren)
+    random_words = %w[lemon pineapple banana cheese orange butterfly stilleven vanitas painting ballet unicorn heart naakt dieren]
     random_word = random_words.sample
-    get_image(random_word).merge({
-      "title" => "Couldn't find an image for that."
-    })
+    get_image(random_word).merge(
+      'title' => "Couldn't find an image for that."
+    )
   end
 
   def get_image(search_term)
-    api_key = ENV["RIJKSMUSEUM_API_KEY"]
-    image = Rijksmus::API::Client.new(search_term, api_key).random_image_search
+    api_key = ENV['RIJKSMUSEUM_API_KEY']
+    Rijksmus::API::Client.new(search_term, api_key).random_image_search
   end
 
   def reply_with_image(tweet, image)
-    image_url = image["webImage"]["url"]
-    image_title = image["title"]
-    screen_name =  "@" + tweet.user.screen_name
-    text = screen_name + " " + image_title
-    @twitter_api.update_with_media(text, open(image_url), in_reply_to_status_id: tweet.id)
+    image_url = image['webImage']['url']
+    image_title = image['title']
+    screen_name = '@' + tweet.user.screen_name
+    text = screen_name + ' ' + image_title
+    @twitter_api.update_with_media(text, open(image_url), in_reply_to_status_id: tweet.id) # rubocop:disable Security/Open
```

Comment:
What does this mean 
"# rubocop:disable Security/Open"

Reply:
Good question! This is a bit tricky.
From parts.

```ruby
# rubocop:disable Security/Open
```

is one way to avoid rubocop to avoid the offense/complain in this specific line*(like .rubocop.yml for root file or specific file but just for a puntual line).
If you take out this comment you will see rubocop  offense in this line with"Security/Open".

Then what is the Security/Open issue?
[Here is the answer](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Security/Open)

Last question and the most  awkward, one.  So, why do we need to disable instead fixing and specify File or IO ?
Answer: File and IO are modules include Ruby language core. But we are actually calling it from 'open-uri' gem #open, the module is URI#open. So. Rubocop is confused about the open name.

Now that you asked about, I think is better to specify the module we use and to delete "# rubocop:disable Security/Open"
```ruby
URI.open(image_url)
```
I am happy to refacto this code. but you also can do it, as you want :).
I check it, it seems that the test still passing rubocop is happy about the change and seems working. but maybe please recheck.

Reply:
Thanks for deep diving into this!